CNAME

```diff
@@ -1 +1 @@
-a11yproject.com
```

Comment:
Yes, I made a mistake with `CNAME` file. But `_config.yml` I think that you need fix it.